src/applications/dhp-connected-devices/tests/dhp-consent-flow.cypress.spec.js

```diff
@@ -14,11 +14,11 @@ describe(manifest.appName, () => {
       },
     };
     cy.intercept('GET', '/v0/feature_toggles*', featureToggles);
+    cy.visit(manifest.rootUrl);
```

Comment:
This needed to be updated for a [similar reason](https://github.com/department-of-veterans-affairs/vets-website/pull/22455/files#r1004005542)